salt/cloud/clouds/vmware.py

```diff
@@ -2350,6 +2393,33 @@ def create(vm_):
             )
             clone_spec.customization = custom_spec
 
+        if customization and (devices and 'network' in list(devices.keys())) and 'Windows' in object_ref.config.guestFullName:
```

Comment:
Line 2396-2400 is redundant. It's already present before for Linux. No need to specify again as the checks can be done within the same If block